docs/user-guide/federation/index.md

```diff
@@ -46,3 +46,26 @@ The following guides explain some of the resources in detail:
 
 [API reference docs](/federation/docs/api-reference/readme/) lists all the
 resources supported by federation apiserver.
+
+## Cascading deletion
+
+Kubernetes version 1.5 includes support for cascading deletion of federated
+resources. With cascading deletion, when you delete a resource from the
+federation control plane, the corresponding resources in all underlying clusters
+are also deleted.
+
+To enable cascading deletion, set the option
+DeleteOptions.orphanDependents=false when you delete a resource from the
```

Comment:
Can you code-format the flag, please? Backticks are fine (DeleteOptions.orphanDependents=false)